zeppelin-interpreter/src/main/java/org/apache/zeppelin/display/Input.java

```diff
@@ -108,7 +108,7 @@ public Input(String name, Object defaultValue, String type, ParamOption[] option
   }
 
   public Input(String name, String displayName, String type, String argument, Object defaultValue,
-      ParamOption[] options, boolean hidden) {
+               ParamOption[] options, boolean hidden) {
```

Comment:
@bitchelov Apache Zeppelin, basically, follows Google Java style guide. You can check it: https://google.github.io/styleguide/javaguide.html#s4.6.3-horizontal-alignment